vcpkg/contributing/maintainer-guide.md

```diff
@@ -12,6 +12,29 @@ It is intended to serve the role of
 [Homebrew's Maintainer Guidelines](https://docs.brew.sh/Maintainer-Guidelines), and
 [Homebrew's Formula Cookbook](https://docs.brew.sh/Formula-Cookbook).
 
+## Overall Registry Design Goals
+
+### Ports in the current baseline must be simultaneiously installable
+
+We wish to be able to show downstream customers of libraries in the default registry that the
+combination of libraries in any given baseline we publish have been tested to work together in at
+least some configurations. Allowing ports to exclude each other breaks the ability to test such
+configurations, as the number of builds necessary for such tests would grow as
+`2^number_of_such_cases`. Moreover, installing additional dependencies is always considered "safe":
+there is no way for a port or end user to assert that a dependency is *not* installed in their
+requirements.
+
+If you wish to represent such an alternative situation for customers, consider describing how
+someone can create an [overlay port](../concepts/overlay-ports.md) implementing the alternative
+form with a comment in `portfile.cmake` rather than trying to add additional ports never built in
+the curated registry's continuous integration. For example, see
+[glad@0.1.36](https://github.com/microsoft/vcpkg/blob/67cc1677c3bf5c23ea14b9d2416c7422fdeac492/ports/glad/portfile.cmake#L17).
+
+Before the introduction of [registries](../maintainers/registries.md), we accepted several
+not tested ports-as-alternatives, such as `boringssl`, that could make authoring overlay ports
+easier. This is no longer accepted because registries allow publishing of these untested ports
+without modifying the curated registry.
+
 ## PR structure
 
 ### Make separate Pull Requests per port

```

Comment:
```suggestion
### Make separate pull requests per port
```
This wasn't changed by your PR, but looking at this, I don't think we need to capitalize "Pull Requests" when we spell out both words.